spec/system/switching_path_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Switching Paths', type: :system do
+  let!(:default_path) { create(:path, title: 'Foundations', default_path: true) }
+  let!(:unselected_path) { create(:path, title: 'Rails') }
+
+  before do
+    sign_in(create(:user, path: default_path))
+  end
+
+  context 'on the paths index page' do
+    it 'allows a user to switch paths' do
+      visit paths_path
+
+      expect(find(:test_id, 'Foundations').value).to eq('Resume')
+      expect(find(:test_id, 'Rails').value).to eq('Switch')
+
+      find(:test_id, 'Rails').click
+
+      expect(page).to have_css '.alert-success'
+      expect(find(:test_id, 'flash')).to have_text 'You have switched to the Rails path'
```

Comment:
What do you think about having an extra assertion here to test the users path has changed in the db?

Reply:
I thought that would be tested at a lower level but if we aren't I can add it

Reply:
Yeah your right, ideally we would have a switch path service where we could test this. But there wasn't enough involved in switching the paths to justify making a service. Testing it here is a good middle ground I reckon.